### PR TITLE
Added missing connectors from condit-labs

### PR DESCRIPTION
**Pull request by uchennakevinm1**
If there are no check boxes on the source and destination columns for a given connector, then that repository has yet to be updated with src/dest code and will be updated at a later time.

### CODE REVIEW

File: docs/connectors.md [hunk 1]
```diff
@@ -28,21 +28,28 @@ Source means the connector has the ability to get data from an upstream data sto
 | Connector | Source | Destination | Support |
 |-----------|-------|----|-------------|
 | [Algolia](https://github.com/ConduitIO/conduit-connector-algolia) | |✅ | Conduit |
```
Comment (lovromazgon) [inline docs/connectors.md:30]: This one was moved to `https://github.com/conduitio-labs/conduit-connector-algolia`.
```suggestion
| [Algolia](https://github.com/conduitio-labs/conduit-connector-algolia) | |✅ | Conduit |
```

File: docs/connectors.md [hunk 2]
```diff
@@ -28,21 +28,28 @@ Source means the connector has the ability to get data from an upstream data sto
...
+| [Airtable](https://github.com/conduitio-labs/conduit-connector-airtable) | | | Community |
```
Comment (lovromazgon) [inline docs/connectors.md:31]: Nitpick: Airtable is alphabetically before Algolia 🙃

Comment (uchennakevinm1) [reply docs/connectors.md:31]: Exactly. Thanks!

**Review by lovromazgon — COMMENTED** [review]
I think we are missing another one - https://github.com/conduitio-labs/conduit-connector-nats-pubsub

**Comment by neovintage** [comment]
@uchennakevinm1 are we waiting to merge this until next week?

**Comment by uchennakevinm1** [comment]
> @uchennakevinm1 are we waiting to merge this until next week?

That right. I want to give Yalantis an opportunity to provide an update on finalizing connectors for v0.3.0.

**Comment by uchennakevinm1** [comment]
@lovromazgon bringing this to your attention

**Review by lovromazgon — APPROVED** [review]
👍

**Pull request merged**